model/src/order.rs

```diff
@@ -314,13 +314,19 @@ impl Display for OrderUid {
         let mut bytes = [0u8; 2 + 56 * 2];
         bytes[..2].copy_from_slice(b"0x");
         // Unwrap because the length is always correct.
-        hex::encode_to_slice(&self.0, &mut bytes[2..]).unwrap();
+        hex::encode_to_slice(self.0, &mut bytes[2..]).unwrap();
```

Comment:
Why this change?

Reply:
This change snuck in when moving the method implementation back and forth. Will remove.